### ISSUE TITLE
So... about security

### ISSUE THREAD

**Issue by esell** [esell/dockopotamus#7]
What are some steps that can be taken to make this a bit more secure? 

* using -cap-drop=all breaks a lot of the container since it's assumed users will want to download/create stuff
* document how to create a docker group so we can avoid running as root
* document how to forward port 22 -> dockopotamus port to avoid running as root
* the main attack vector seems to be the docker daemon, what do we do about that?

**Comment by nicolasdanelon** [comment]
what if you add your user to the docker group? why people run docker as root :/

**Comment by esell** [comment]
Yes @nicolasdanelon, I think what would be best would be to include these steps inside of the readme or in a wiki page. Basically provide a "best practices" for running this which would include creating a docker group, setting up iptables so that the process doesn't have to bind to port 22, etc.
